charts/jaeger/templates/cassandra-schema-job.yaml

```diff
@@ -46,10 +46,8 @@ spec:
             secretKeyRef:
               name: {{ if .Values.storage.cassandra.existingSecret }}{{ .Values.storage.cassandra.existingSecret }}{{- else }}{{ include "jaeger.fullname" . }}-cassandra{{- end }}
               key: password
-      {{- if .Values.schema.keyspace }}
         - name: KEYSPACE
-          value: {{ .Values.schema.keyspace | quote }}
-      {{- end }}
+          value: {{ .Values.storage.cassandra.keyspace | quote }}
```

Comment:
Please include in a seperate PR describing the problem this solves as it is not immediately obvious to me! Thanks

Reply:
Here is the discussion https://github.com/jaegertracing/jaeger-operator/pull/837